CHANGES.rst

```diff
@@ -64,6 +143,18 @@ resample
   a bug due to which parts of input images may not be present in the output
   resampled image under certain circumstances. [#7460]
 
+- Carry through good bits correctly for the variance array [#7515]
+
+residual_fringe
+---------------
+
+- Fix bug in Residual Fringe step handling of NaNs in input [#7471]
+
+regtest
+-------
```

Comment:
This needs to be moved up the top section, for 1.10.1 unreleased, because it wasn't part of 1.10.0.